cvat/apps/events/handlers.py

```diff
@@ -141,7 +135,7 @@ def user_name(instance=None):
 
 def user_email(instance=None):
     current_user = get_user(instance)
-    return _get_value(current_user, "email")
+    return _get_value(current_user, "email") or None
```

Comment:
Email may be empty string in our database.
EventSerializer does not accept blank field (empty string). Only null or nothing. 
It will throw validation error. 

Usually we do not run event.is_valid() for events generated on the server, but I did it for working time event and this error occurred

But yes, this change does not affect working time logic

Reply:
Okay (although IMO, email being an empty string in the DB is a bug).